apps/backend/src/Gump.Data/Repositories/CategoryRepository.cs

```diff
@@ -0,0 +1,100 @@
+using Gump.Data.Models;
+using MongoDB.Bson;
+using MongoDB.Driver;
+
+namespace Gump.Data.Repositories;
+
+public class CategoryRepository : RepositoryBase<CategoryModel>
+{
+	public CategoryRepository(string connectionString) : base(connectionString)

```

Comment:
Make this 1 sor. A kapcsoszárójelek eférnek ennek a sornak a végén

Reply:
miért nem csinálja meg az editorconfig??